CONTRIBUTING.md

```diff
@@ -94,12 +94,18 @@ We use the following components in a WGPU development environment:
 
 - [The version of the Rust toolchain with the `cargo` command][install-rust],
   pointed to by `rust-toolchain.toml` at the root of the repository, to compile
-  WGPU's code.
+  WGPU's code. If you use rustup, this will be automatically installed when you
+  first run a `cargo` command in the repository.
```

Comment:
nitpick: backticks on `rustup` plz

thought: This requirement description feels…circuitous.

thought: It'd be nice to link to `rust-toolchain.toml` here.

suggestion: All things combined:

```suggestion
- [A Rust toolchain][install-rust] matching the version specified in
  [`rust-toolchain.toml`](./rust-toolchain.toml), to compile WGPU's code. If you
  use `rustup`, this will be automatically installed when you first run a
  `cargo` command in the repository.
```

Reply:
Fixing with latest push.